pages/index.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+  <div>
+    <h1>Home page</h1>
+    <button @click="goToPage('/team/about')">Team section</button>
+    <button @click="goToPage('/project/login')">Project section</button>
```

Comment:
Router instance is auto-injected in nuxt, you can use `$router.push(...)` instead of custom function